process.c

```diff
@@ -3020,77 +3005,67 @@ NORETURN(static VALUE f_exec(int c, const VALUE *a, VALUE _));
 
 /*
  *  call-seq:
- *     exec([env,] command... [,options])
- *
- *  Replaces the current process by running the given external _command_, which
- *  can take one of the following forms:
+ *    exec([env, ] command_line, options = {})
+ *    exec([env, ] exe_path, options  = {})
+ *    exec([env, ] exe_path, *args, options  = {})
  *
- *  [<code>exec(commandline)</code>]
- *	command line string which is passed to the standard shell
- *  [<code>exec(cmdname, arg1, ...)</code>]
- *	command name and one or more arguments (no shell)
- *  [<code>exec([cmdname, argv0], arg1, ...)</code>]
- *	command name, +argv[0]+ and zero or more arguments (no shell)
+ *  Replaces the current process by doing one of the following:
  *
- *  In the first form, the string is taken as a command line that is subject to
- *  shell expansion before being executed.
+ *  - Passing string +command_line+ to the shell.
+ *  - Invoking the executable at +exe_path+.
  *
- *  The standard shell always means <code>"/bin/sh"</code> on Unix-like systems,
- *  otherwise, <code>ENV["RUBYSHELL"]</code> or <code>ENV["COMSPEC"]</code> on
- *  Windows and similar.  The command is passed as an argument to the
- *  <code>"-c"</code> switch to the shell, except in the case of +COMSPEC+.
+ *  The new process is created using the
+ *  {exec system call}[https://www.man7.org/linux/man-pages/man1/exec.1p.html];
```

Comment:
That page is not the system call.
https://pubs.opengroup.org/onlinepubs/9699919799.2018edition/functions/execve.html

Reply:
Fixed.